src/pages/About/index.tsx

```diff
@@ -0,0 +1,240 @@
+import coinbaseWalletIcon from 'assets/images/coinbaseWalletIcon.svg'
+import metaMaskIcon from 'assets/images/metamask.png'
+import phantomIcon from 'assets/images/phantom.png'
+import rainbowIcon from 'assets/images/rainbow.png'
+import walletConnectIcon from 'assets/images/walletConnectIcon.svg'
+import { ButtonOutlined } from 'components/Button'
+import { Box } from 'nft/components/Box'
+import { DiscordIconMenu, GithubIconMenu, TwitterIconMenu } from 'nft/components/icons'
+import { ReactNode } from 'react'
+import { useIsDarkMode } from 'state/user/hooks'
+import styled, { useTheme } from 'styled-components/macro'
+import { BREAKPOINTS } from 'theme'
+
+import Card from './Card'
+import Step from './Step'
+import { SubTitle, Title } from './Title'
+
+const MOBILE_BREAKPOINT = BREAKPOINTS.md
+
+const IconRow = styled.span`
+  display: flex;
+  flex-direction: row;
+`
+
+const Icon = ({ href, children }: { href?: string; children: ReactNode }) => {
+  return (
+    <Box
+      as={href ? 'a' : 'div'}
+      href={href ?? undefined}
+      target={href ? '_blank' : undefined}
+      rel={href ? 'noopener noreferrer' : undefined}
+      display="flex"
+      flexDirection="column"
+      color="textPrimary"
+      background="none"
+      border="none"
+      justifyContent="center"
+      textAlign="center"
+      marginRight="12"
+    >
```

Comment:
Can you solve this w/o importing the `Box` component from `nft/components`? maybe use `styled(ExternalLink)` instead, since that provides a lot of the config here?

Reply:
yea this was a copy paste but ill change it

Reply:
i need to change this out soon for a new design so will do in a new pr